exporter/prometheusremotewriteexporter/wal.go

```diff
@@ -236,6 +246,28 @@ func (prweWAL *prweWAL) run(ctx context.Context) (err error) {
 	return nil
 }
 
+func (prweWAL *prweWAL) recordLagLoop(ctx context.Context, logger *zap.Logger) {
+	ticker := time.NewTicker(15 * time.Second)
```

Comment:
Could you add the interval period to the `prweWAL` struct? This way, we can override the period to a few milliseconds during tests.